### ISSUE TITLE
Crash at startup

### ISSUE THREAD

**Issue by stijnvn** [iguissouma/nx-console-idea-plugin#3]
The plugin (v0.7.0) crashes with the following error:

```
java.lang.NullPointerException: null cannot be cast to non-null type com.intellij.json.psi.JsonObject
	at com.github.iguissouma.nxconsole.buildTools.NxJsonUtil.doBuildStructure(NxJsonUtil.kt:80)
	at com.github.iguissouma.nxconsole.buildTools.NxJsonUtil.access$doBuildStructure(NxJsonUtil.kt:21)
	at com.github.iguissouma.nxconsole.buildTools.NxJsonUtil$listTasks$1.run(NxJsonUtil.kt:47)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:835)
	at com.github.iguissouma.nxconsole.buildTools.NxJsonUtil.listTasks(NxJsonUtil.kt:45)
	at com.github.iguissouma.nxconsole.buildTools.NxService.hasScripts(NxService.kt:115)
	at com.github.iguissouma.nxconsole.buildTools.NxService.access$hasScripts(NxService.kt:41)
	at com.github.iguissouma.nxconsole.buildTools.NxService$detectAllBuildfilesInContentRoots$1.consume(NxService.kt:106)
	at com.github.iguissouma.nxconsole.buildTools.NxService$detectAllBuildfilesInContentRoots$1.consume(NxService.kt:41)
	at com.intellij.lang.javascript.buildTools.base.JsbtUtil.iterateOverContentRoots(JsbtUtil.java:76)
	at com.github.iguissouma.nxconsole.buildTools.NxService.detectAllBuildfilesInContentRoots(NxService.kt:103)
	at com.github.iguissouma.nxconsole.buildTools.NxJsbtStartupActivity$Companion$doDetect$1.run(NxJsbtStartupActivity.kt:61)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:840)
	at com.github.iguissouma.nxconsole.buildTools.NxJsbtStartupActivity$Companion.doDetect(NxJsbtStartupActivity.kt:57)
	at com.github.iguissouma.nxconsole.buildTools.NxJsbtStartupActivity$Companion.access$doDetect(NxJsbtStartupActivity.kt:17)
	at com.github.iguissouma.nxconsole.buildTools.NxJsbtStartupActivity$Companion$scheduleDetection$1$1$1.run(NxJsbtStartupActivity.kt:41)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:20)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:11)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.call(ApplicationImpl.java:268)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Any idea what could be wrong?

**Comment by iguissouma** [comment]
Hello,

it seems to be a problem with your `nx.json` file.
Can you attach your `nx.json` file?

**Comment by iguissouma** [comment]
issue solved in version 0.8.0
